autoload/LanguageClient.vim

```diff
@@ -201,6 +209,15 @@ function! s:inputlist(...) abort
     return l:selection
 endfunction
 
+function! s:selectionUI_funcref(source, sink) abort
+    if get(g:, 'LanguageClient_selectionUI', v:null) ==? 'FZF'
+                \ || get(g:, 'loaded_fzf')
```

Comment:
I don't think this is what you want here. As it stands, if `loaded_fzf` is set then fzf will be used, regardless of what you have configured in `selectionUI`.

I think I would do something like `if get(g:, 'LanguageClient_selectionUI', 'FZF')  ==? 'FZF' && get(g: 'loaded_fzf')` instead, that way FZF will be the default but it will only be used if it's loaded. And if `selectionUI` is another thing, different than FZF, it will go to the else branch.